android/brave_java_resources.gni

```diff
@@ -402,6 +377,7 @@ brave_java_resources = [
   "java/res/drawable/ads_custom_notification_border_dark.xml",
```

Comment:
one thought on this 
this dark file can remove and update color code both default and night brave_color.xml Like 
 in  "java/res/drawable/ads_custom_notification_border.xml",
<solid android:color="@color/custom_notification_solid_color" />
   <stroke android:width="1dip" android:color="@color/custom_notification_storke_color"/>
And update solid_color and stroke_color both in default and night brave_color

It will not make any big difference but will have one file.